lib/stdlib/test/epp_SUITE.erl

```diff
@@ -113,6 +113,16 @@ include_local(Config) when is_list(Config) ->
         [ FileLine || {attribute,_,file,FileLine} <- List ],
     ok.
 
+file_macro(Config) when is_list(Config) ->
+    DataDir = proplists:get_value(data_dir, Config),
+    File = filename:join(DataDir, "file_macro.erl"),
+    {ok, List} = epp:parse_file(File, [DataDir], [{source_name, "Other source"}]),
+    %% Both attribute a and b are defined as ?FILE, they should be the same
+    {value, {attribute,_,a,FileA}} = lists:keysearch(a,3,List),
+    {value, {attribute,_,b,FileB}} = lists:keysearch(b,3,List),
+    FileA = FileB,
```

Comment:
Why only check that they're the same? More explicit would be to also check that they're the expected value:

```suggestion
    "Other source" = FileA = FileB,
```

Reply:
That's a great idea!